packages/cra-template-seed/template/src/components/Shell/Shell.js

```diff
@@ -0,0 +1,61 @@
+import React, { useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+
+import { ShellBar } from '@ui5/webcomponents-react/lib/ShellBar';
+import { Avatar } from '@ui5/webcomponents-react/lib/Avatar';
+import { AvatarShape } from '@ui5/webcomponents-react/lib/AvatarShape';
+import { AvatarSize } from '@ui5/webcomponents-react/lib/AvatarSize';
+import BrowserProvider from '../../util/browser/BrowserProvider';
+import PopoverListItems from '../Popover/List/PopoverListItems';
+import ThemeSwitch from '../ThemeSwitch/ThemeSwitch';
+
+const style = {
+  shell: {
+    position: 'fixed',
+    width: '100%',
+    zIndex: 100,
+  },
+  emptySpace: {
+    paddingTop: '44px',
+  },
+};
+
+const Shell = ({ title, ...props }) => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const popoverConfigItemsRef = useRef(null);
+  const themeSwitchRef = useRef(null);
+  const popoverItems = [
```

Comment:
useMemo in order to keep the ref stable?

Reply:
What do you mean by adding useMemo?